packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -200,7 +246,15 @@ export const getPublicEvent = async (
     // Sets user data on profile object for easier access
     profile: getProfileFromEvent(event),
     users,
-    orgDomain: org,
+    entity: {
+      isUnpublished:
+        event.team?.slug === null ||
+        event.owner?.organization?.slug === null ||
+        event.team?.parent?.slug === null,
+      orgSlug: org,
+      teamSlug: (event.team?.slug || teamMetadata?.requestedSlug) ?? null,
```

Comment:
`orgSlug` and `teamSlug` are defined separately to know which entity is being accessed, in case both are defined is because a sub-team is being accessed